frontend/src/components/CreateGame.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div id="create-game">
+    </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  name: "CreateGame",
+  beforeRouteEnter: async function(to, from, next) {
+    try {
+      const response = await axios.post("http://backend:3000/games");
```

Comment:
I assume somehow this calls core.api/add-player
Would it be possible to comment that so it's more understandable for us plebeians?

Reply:
The interface is defined in card-game.api-handler (inside the `(defroutes app-routes` object)